design-patterns/14_Command.ipynb

```diff
@@ -0,0 +1,267 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Command Design Pattern\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## The Command Pattern encapsulates a request as an object, thereby letting you parameterize other objects\n",
+    "## with different requests, queue or log requests, and support undoable operations.\n",
+    "\n",
+    "![title](Pictures/command_design_pattern.png)\n"
```

Comment:
Also, not sure how you gonna embed your local pictures to notebook?

Reply:
you can use link locations from actual pictures